### PR TITLE
Work on shelly request list 

### PR DESCRIPTION
**Pull request by nivcertora**
This PR aims to complete the following requests list : 

`
https://docs.certora.com/en/latest/docs/user-guide/faq.html?highlight=incomplete - let's remove this page
https://docs.certora.com/en/latest/docs/user-guide/getting-started/layout.html?highlight=incomplete - to be removed
https://docs.certora.com/en/latest/docs/user-guide/patterns/general.html?highlight=incomplete - to be removed
https://docs.certora.com/en/latest/docs/cvl/sorts.html?highlight=incomplete - convert from old docs
https://docs.certora.com/en/latest/docs/cvl/sorts.html?highlight=incomplete - convert from old docs
https://docs.certora.com/en/latest/docs/prover/cli/linking.html?highlight=incomplete#linking - convert from old docs
https://docs.certora.com/en/latest/docs/user-guide/troubleshooting.html?highlight=incomplete#troubleshooting - to be removed
https://docs.certora.com/en/latest/docs/cvl/defs.html?highlight=incomplete#definitions - convert from old docs
https://docs.certora.com/en/latest/docs/prover/approx/summarization.html?highlight=incomplete#method-summarization - convert from old docs
https://docs.certora.com/en/latest/docs/prover/cli/verification.html?highlight=incomplete#verification - keep just "Certora Prover CLI Options"
https://docs.certora.com/en/latest/docs/prover/cli/verification.html?highlight=incomplete#verification - easy to fix
https://docs.certora.com/en/latest/docs/prover/portal/report.html?highlight=incomplete#certora-verification-reports - we can add a section on reverts, havocs, call resolution table
https://docs.certora.com/en/latest/docs/prover/portal/report.html?highlight=incomplete#certora-verification-reports - convert from old docs
https://docs.certora.com/en/latest/docs/prover/portal/report.html?highlight=incomplete#certora-verification-reports - convert from old docs. and maybe mention one example where functions are indeed more powerful than mappings (transitive reachability?)
https://docs.certora.com/en/latest/docs/user-guide/properties/index.html?highlight=incomplete#designing-good-specifications - remove the whole section
https://docs.certora.com/en/latest/docs/cvl/using.html?highlight=incomplete#using-statements - convert from old docs and update as necessary
https://docs.certora.com/en/latest/docs/cvl/hooks.html?highlight=incomplete#hooks - convert from old docs
https://docs.certora.com/en/latest/docs/cvl/functions.html?highlight=incomplete#syntax - convert from old docs
https://docs.certora.com/en/latest/docs/cvl/statements.html?highlight=incomplete#requireinvariant-statements - write
https://docs.certora.com/en/latest/docs/cvl/statements.html section on Solidity-like statements, function calls - write
https://docs.certora.com/en/latest/docs/cvl/statements.html?highlight=incomplete#requireinvariant-statements - convert from old docs
https://docs.certora.com/en/latest/docs/user-guide/getting-started/running.html?highlight=incomplete#running-the-certora-prover - write
https://docs.certora.com/en/latest/docs/user-guide/checking.html?highlight=incomplete#checking-specifications - write
https://docs.certora.com/en/latest/docs/prover/approx/harnessing.html?highlight=undocumented - remove munging complete. show an example for harnessing and why it is absolutely necessary
https://docs.certora.com/en/latest/docs/cvl/imports.html?highlight=undocumented#import-and-use-statements - can probably convert from old docs or just write it
https://docs.certora.com/en/latest/docs/cvl/expr.html?highlight=undocumented#basic-operations - document >>>
https://docs.certora.com/en/latest/docs/cvl/expr.html?highlight=undocumented#calling-contract-functions - we removed @dontsummarize
https://docs.certora.com/en/latest/docs/cvl/expr.html?highlight=undocumented#extended-logical-operations - write on short-circuiting
https://docs.certora.com/en/latest/docs/user-guide/patterns/sums.html?highlight=todo#tracking-sums - write
https://docs.certora.com/en/latest/docs/user-guide/patterns/states.html?highlight=todo#writing-state-based-invariants - write
https://docs.certora.com/en/latest/docs/confluence/advanced/linking.html?highlight=todo#linking-contracts - old docs, can remove
https://docs.certora.com/en/latest/docs/confluence/advanced/multicontract.html?highlight=todo#multicontract - old docs can remove
https://docs.certora.com/en/latest/docs/user-guide/patterns/partial-apply.html?highlight=todo#partially-parametric-rules - complete
https://docs.certora.com/en/latest/docs/prover/intro.html?highlight=todo#introduction - can remove the todo
https://docs.certora.com/en/latest/docs/user-guide/patterns/safe-assum.html?highlight=todo#listing-safe-assumptions - complete
https://docs.certora.com/en/latest/docs/confluence/advanced/approximation.html?highlight=todo#the-problem - two todos here, should be easy to handle or just remove problematic sentences
https://docs.certora.com/en/latest/docs/user-guide/intro.html?highlight=todo#introduction - can remove the todo
https://docs.certora.com/en/latest/docs/cvl/hooks.html?highlight=todo#syntax - remove the todo. not sure the grammars are useful actually, examples are better
https://docs.certora.com/en/latest/docs/user-guide/multicontract/index.html?highlight=todo#the-dangers-of-dispatcher - put the link to requireInvariant page after it's written
`

### CODE REVIEW

File: docs/cvl/using.md [hunk 1]
```diff
@@ -6,9 +6,7 @@ The `using` statement introduces a variable that can be used to call methods on
 contracts other than the main contract being verified.
 
 ```{todo}
```
Comment (shellygr) [inline docs/cvl/using.md:8]: remove this todo entirely

File: docs/prover/cli/verification.md
Comment (shellygr) [inline docs/prover/cli/verification.md:1]: remove this page completely

**Review by shellygr — COMMENTED** [review]
a bit more removals requested

File: docs/cvl/using.md (hunk 1 above)
Comment (nivcertora) [reply docs/cvl/using.md]: Done

File: docs/prover/cli/verification.md
Comment (nivcertora) [reply docs/prover/cli/verification.md:1]: Done

**Review by shellygr — APPROVED** [review]

**Pull request merged**